plus/api/src/main/java/org/sakaiproject/plus/api/model/Tenant.java

```diff
@@ -91,6 +91,16 @@ public class Tenant extends BaseLTI implements PersistableEntity<String> {
 	@Column(name = "VERBOSE")
 	private Boolean verbose = Boolean.FALSE;
 
+	@Column(name = "SITE_TEMPLATE", length = 500, nullable = true)
```

Comment:
do you need to use 500 here? It will generate a big type in mysql I think @ern ?

Reply:
if it's a site id that goes here, it should be 99 to match

Reply:
Yeah - I already have a predefined length for a sakai site id in my parent class - switched - thanks.